content/microcontrollers/particle-argon.md

```diff
@@ -15,6 +15,7 @@ weight: 3
 | I2C      | YES | YES |
 | ADC      | YES | YES |
 | PWM      | YES | YES |
+| Bluetooth      | YES | Not yet |
```

Comment:
There is `-target=particle-argon-s140v7`.

Reply:
How do you flash the SoftDevice for it? We can add this info.

Reply:
It should be identical to how you flash the microbit or reel board.

Reply:
@suda are you around? Can you chime in as far as how we can flash the SoftDevice/TinyGo on the Particle boards? Thank you!

Reply:
I don't know if I have the capacity ATM but I sent some Argons to Ayke and Daniel a while ago so I hope they could take a stab maybe?

Reply:
I could re-send them to someone else. Not having much time at the moment

Reply:
Sure, we can talk private message about that. Thanks!